src/app/static/src/tests/components/header/hintrVersionMenu.test.ts

```diff
@@ -34,6 +36,19 @@ describe("Hintr Menu Version", () => {
         expect(wrapper.findAll("span").length).toBe(5);
     });
 
+    it("hintr version menu displays link to news site", async() => {
+        const store = createStore();
+        const wrapper = shallowMount(HintrVersionMenu, {
+            store
+        });
+
+        const links = wrapper.findAll("a");
+        expect(links.length).toBe(1);
+        const link = links.at(0);
+        expect(link.attributes("href")).toBe("https://naomi.unaids.org/news");
+        expect(link.attributes("target")).toBe("_blank");
```

Comment:
`expectTranslated(link, "News", "Nouvelles", "Notícias", store)`

You will need to test translated text.